src/app/components/profile/profile.component.html

```diff
@@ -0,0 +1,31 @@
+<mat-card>
+  <mat-card-header>
+    <mat-card-title>Name</mat-card-title>
```

Comment:
please use the general title for all user data ("Profile" for example)

Reply:
Done